deploy/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Compiler container
+FROM node:12-alpine AS compiler
+RUN apk add --no-cache build-base git bash
+WORKDIR /usr/src/app
+COPY package.json ./
+RUN npm i --cache /tmp/1
+COPY . ./
```

Comment:
Are we sure it works? Because in my understanding (not so deep), here we are copying the content of the `deploy` folder, so I'm afraid it doesn't work.